smriprep/workflows/surfaces.py

```diff
@@ -532,9 +537,17 @@ def init_gifti_surface_wf(*, name="gifti_surface_wf"):
         run_without_submitting=True,
     )
     fs2gii = pe.MapNode(
-        fs.MRIsConvert(out_datatype="gii", to_scanner=True), iterfield="in_file", name="fs2gii"
+        fs.MRIsConvert(out_datatype="gii", to_scanner=True), iterfield="in_file", name="fs2gii",
+    )
+    surfmorph_list = pe.Node(
+        niu.Merge(3, ravel_inputs=True),
+        name="surfmorph_list",
+        run_without_submitting=True,
+    )
+    fscurv2funcgii = pe.MapNode(
+        MRIsConvertData(out_datatype="gii", to_scanner=True, target_surface="smoothwm"),
+        iterfield="scalarcurv_file", name="fscurv2funcgii",
     )
-    fix_surfs = pe.MapNode(NormalizeSurf(), iterfield="in_file", name="fix_surfs")
```

Comment:
Why are we dropping `fix_surfs`? See #295,

Reply:
No. These are updates to the coordinates themselves. Curvature files are just scalar maps.